app/src/pages/inside/projectSettingsPageContainer/content/analyzerContainer/autoAnalysis/autoAnalysis.jsx

```diff
@@ -107,15 +107,6 @@ const AutoAnalysis = ({
         data[ANALYZER_MODE],
       ),
     );
-
-    const numberOfLogLines = data[NUMBER_OF_LOG_LINES] === '-1' ? 'all' : data[NUMBER_OF_LOG_LINES];
-
-    trackEvent(
-      PROJECT_SETTINGS_ANALYZER_EVENTS.clickSubmitInIndexTab(
-        numberOfLogLines,
-        data[ALL_MESSAGES_SHOULD_MATCH],
-      ),
-    );
```

Comment:
I'm not sure, but may be we need to track this parameter together with `clickSubmitInAutoAnalyzerTab`.
Should be discussed with QA & GA.

Reply:
Done.